### DISCUSSION TITLE
0.10.16

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#8468] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/0.10.16'>0.10.16</a>.</em>

**Comment by flarive** [comment]
Hi Dan,

Could you please publish the change log ?
Thanks a lot for this cool new release :)
I'm going to use it asap in Neumorphism.Avalonia !
https://github.com/flarive/Neumorphism.Avalonia

**Reply by maxkatz6** [reply]
Hi flarive,
I was busy yesterday, could add changelog in time.
Published changelog now.
